addon/v1alpha1/types_managedclusteraddon.go

```diff
@@ -142,6 +147,25 @@ type ObjectReference struct {
 	Name string `json:"name"`
 }
 
+type HealthCheckMode string
```

Comment:
nit: enumerize

Reply:
yeah, add the enum comment

Reply:
I think you need to add kubebuilder's enum marker

Reply:
use kubebuilder marker